x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_list_page.tsx

```diff
@@ -242,7 +243,7 @@ export const ArtifactListPage = memo<ArtifactListPageProps>(
           />
         )}
 
-        {!doesDataExist ? (
+        {!items.length ? (
```

Comment:
This is not correct. I needs to be the `doesDataExist` which is different from `items`. The content of `items` indicates if there is any data to be displayed given the current filters - and that could be 0. In those cases, we don't show the empty state on the page, we just show the normal list page with "no results" type of message. 

I'm wondering if perhaps you hit a bug in the logic around `doesdataExist` when going from "no data exists", to creating your first item to then the empty state continuing to be shown. I'll review that that area of code again

Reply:
Ok... Just based on my code comment, looks like there is a bug in the logic around `doesDataExist`

https://github.com/elastic/kibana/blob/567512e6fe01f09b600d7d1125606c1afd863ffe/x-pack/plugins/security_solution/public/management/components/artifact_list_page/hooks/use_with_artifact_list_data.ts#L120-L126

I'll fix it in my PR. for now, you can get around it by just ensuring you have at least one item created then revisit the list and it should show

Reply:
FYI - just pushed a fix for this in my PR here: d895c30452e316169ece5ef7c77bff0c63941fe1

Reply:
👍  will remove this then.